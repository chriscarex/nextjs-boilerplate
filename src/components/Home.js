import Link from 'next/link';
import { ROUTE_NEXT } from '../constants/routes';

const Home = () => (
  <div>
    <h1>
      Welcome to the NextJs boilerplate!
    </h1>

    <div>
      <Link href={ROUTE_NEXT}>
        <a>Click here to go to the next page</a>
      </Link>
    </div>

  </div>
);

export default Home;
