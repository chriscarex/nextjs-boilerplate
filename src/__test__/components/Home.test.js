import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../../components/Home';

const setup = () => {
  const props = {};

  const component = shallow(<Home {...props} />);

  return {
    component,
    props
  };
};

describe('Home', () => {
  it('should match snapshot ', () => {
    const {
      component
    } = setup();

    expect(toJson(component)).toMatchSnapshot();
  });
});
