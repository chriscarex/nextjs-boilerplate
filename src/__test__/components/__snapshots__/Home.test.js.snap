// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot  1`] = `
<div>
  <h1>
    Welcome to the NextJs boilerplate!
  </h1>
  <div>
    <Link
      href="/next"
    >
      <a>
        Click here to go to the next page
      </a>
    </Link>
  </div>
</div>
`;
